components/Companies.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+const sites = [
+  {
+    src: '/logos/logo-Checkr.svg',
+    alt: 'Checkr logo',
+    url: 'https://docs.temporal.io/blog/how-temporal-simplified-checkr-workflows'
+  },
+  {
+    src: '/logos/logo-Coinbase.svg',
+    alt: 'Coinbase logo',
+    url: 'https://docs.temporal.io/blog/reliable-crypto-transactions-at-coinbase'
+  },
+  {
+    src: '/logos/logo-Box.svg',
+    alt: 'Box logo',
+    url: 'https://docs.temporal.io/blog/Temporal-a-central-brain-for-Box'
+  },
+  {
+    src: '/logos/logo-Descript.svg',
+    alt: 'Descript logo',
+    url: 'https://docs.temporal.io/blog/descript-case-study'
+  }
+];
```

Comment:
can you also add zebra medical vision and airbyte logos pls

```suggestion
  {
    src: '/logos/logo-Descript.svg',
    alt: 'Descript logo',
    url: 'https://docs.temporal.io/blog/descript-case-study'
  },
  {
    src: '/logos/logo-Zebra.svg',
    alt: 'Zebra logo',
    url: 'https://docs.temporal.io/blog/zebra-medical-case-study'
  },
  {
    src: '/logos/logo-Airbyte.svg',
    alt: 'Airbyte logo',
    url: 'https://docs.temporal.io/blog/airbyte-case-study'
  }
];
```

Reply:
Sure! 👍🏻